src/h5web/visualizations/heatmap/utils.ts

```diff
@@ -12,6 +18,10 @@ export function generateCSSLinearGradient(
   return `linear-gradient(to ${direction},${gradientColors})`;
 }
 
+export function getDataScaleFn(isLog: boolean): DataScaleFn {
+  return isLog ? scaleSymlog : scaleLinear;
+}
```

Comment:
I decided to move `getDataScaleFn` here because I don't think it makes sense to use the same logic for both data axis scales and heatmap data scales. The logic to choose the log scale function for a data axis may become more complex because of log/symlog, while the heatmap data scale function may remain simple.

I removed the original `getDataScaleFn` for now, as you'll see, as it was no longer used in multiple places. It may make sense to re-extract it once we implement proper log scale support, but we'll see about it then.

Reply:
>  I don't think it makes sense to use the same logic for both data axis scales and heatmap data scales

I don't fully agree with this but I get your point. As you said, because proper log scale might come, it can stay like this.